app/include/dt-bindings/zmk/all.h

```diff
@@ -0,0 +1,17 @@
+/*
```

Comment:
On the naming here... Not in love with `all.h`. maybe `std_params.h` or just `params.h`?